server/index.js

```diff
@@ -167,7 +211,7 @@ app.post('/api/itinerary', (req, res, next) => {
     .catch(err => next(err));
 });
 
-app.patch('/api/itinerary/:itineraryId', (req, res, next) => {
+app.patch('/api/itinerary/:itineraryId', authorizationMiddleware, (req, res, next) => {
```

Comment:
Your routes that require authorization should also be using the `req.user.userId` in their `where` clauses.